modules/apps/29-fee/client/cli/tx.go

```diff
@@ -1,3 +1,99 @@
 package cli
 
-// TODO
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/ibc-go/v3/modules/apps/29-fee/types"
+	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+)
+
+const (
+	flagRecvFee    = "recv-fee"
+	flagAckFee     = "ack-fee"
+	flagTimeoutFee = "timeout-fee"
+)
+
+// NewPayPacketFeeAsyncTxCmd returns the command to create a MsgPayPacketFeeAsync
+func NewPayPacketFeeAsyncTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "pay-packet-fee [src-port] [src-channel] [sequence]",
+		Short:   "Pay a fee to incentivize an existing IBC packet",
+		Long:    strings.TrimSpace(`Pay a fee to incentivize an existing IBC packet.`),
+		Example: fmt.Sprintf("%s tx pay-packet-fee [src-port] [src-channel] [sequence] --recv-fee 10stake --ack-fee 10stake --timeout-fee 10stake", version.AppName),
```

Comment:
The example should actually have example packet id rather than parameterizing them correct?

Reply:
I don't think any of our cli commands do that. It sounds like a good idea to me